config/300-route.yaml

```diff
@@ -30,3 +30,13 @@ spec:
     shortNames:
     - rt
   scope: Namespaced
+  additionalPrinterColumns:
+  - name: Domain
+    type: string
+    JSONPath: .status.domain
+  - name: Ready
+    type: string
+    JSONPath: ".status.conditions[?(@.type==\"Ready\")].status"
```

Comment:
Can we add a column for the "Reason" associated with the "Ready" condition too?

Reply:
Yep, done.